ios/RNShare.m

```diff
@@ -84,6 +85,7 @@ - (NSDictionary *)constantsToExport
     @"INSTAGRAMSTORIES": @"instagramstories",
     @"TELEGRAM": @"telegram",
     @"EMAIL": @"email",
+    @"MESSENGER": @"messanger",
```

Comment:
fyi, typo here causes failure to access messenger sharing if someone is using the enum exported in their shareSingle method